features/lima/README.md

```diff
@@ -4,24 +4,12 @@
 This feature flag produces an image suitable for using with [lima](https://lima-vm.io)
 </website-feature>
 
-Build the image with `./build kvm-lima`
-
-Example config (build image locally, adapt path, save config as `gardenlinux.yaml`):
-
-```
-images:
-- location: "path/to/kvm-lima-amd64-today-local.qcow2"
-  arch: "x86_64"
-- location: "path/to/kvm-lima-arm64-today-local.qcow2"
-  arch: "aarch64"
-```
-
-Example usage:
+Example usage with a pre-built image:
```

Comment:
suggestion: add a section here, or add a markdown file in `docs/01_developers/lima.md`  and show example/default usage of `features/lima/generate-manifest.py`

Reply:
Will be done in a follow up pr